tools/export/makefile/Makefile.tmpl

```diff
@@ -55,7 +55,12 @@ PROJECT := {{name}}
 {% endfor %}
 LIBRARY_PATHS :={% for p in library_paths %} {{user_library_flag}}{{p}} {% endfor %}
 LIBRARIES :={% for lib in libraries %} {{lib}} {% endfor %}
+
+# Allow user overrides of LINKER_SCRIPT from the command line, useful
+# if user wants to reserve specific flash areas for production config data
+ifndef $(LINKER_SCRIPT)
 LINKER_SCRIPT := {{linker_script}}
+endif
```

Comment:
Let's not reinvent the wheel here. this whole block is equal to
```make
LINKER_SCRIPT ?= {{linker_script}}
```
https://www.gnu.org/software/make/manual/make.html#Setting